permissions/plugin-pipeline-cloudwatch-logs.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: "pipeline-cloudwatch-logs"
+github: "jenkinsci/pipeline-cloudwatch-logs-plugin"
```

Comment:
Though I tend to think this should probably be waiting for the rename to happen before we merge and check this, but well.

Reply:
+1

Reply:
Well, at least for https://github.com/jenkinsci/pipeline-log-fluentd-cloudwatch-plugin/pull/8 to be merged. At that point we are committing to a Fluentd-less system. Then I need to do some boring stuff: renaming the repo, creating the wiki page, changing the `artifactId`, changing package names.